src/_utils.js

```diff
@@ -563,18 +564,39 @@ export const processCss = (stylesInfo, options) => {
     expressions,
     dynamic,
     location,
-    fileInfo,
+    file,
     isGlobal,
     plugins,
-    vendorPrefixes
+    vendorPrefixes,
+    sourceMaps
   } = stylesInfo
 
+  const fileInfo = {
+    code: file.code,
+    sourceRoot: file.opts.sourceRoot,
+    filename: file.opts.filename || file.filename
+  }
+
+  fileInfo.sourceFileName =
+    file.opts.sourceFileName ||
+    file.sourceFileName ||
+    // According to https://babeljs.io/docs/en/options#source-map-options
+    // filenameRelative = path.relative(file.opts.cwd, file.opts.filename)
+    // sourceFileName = path.basename(filenameRelative)
+    (fileInfo.filename &&
```

Comment:
@ijjk what do you think about this?

Reply:
This seems more stable than trying to follow where it's at in the options. Doesn't `path.basename` just return the last part of the path so the `path.relative` isn't necessary inside of the `path.basename` right?

Reply:
👍 indeed.